docs/features/configuration.rst

```diff
@@ -394,7 +398,10 @@ HTTP/2 version policy
     "DownstreamScheme": "https",
     "DownstreamHttpVersion": "2.0",
     "DownstreamHttpVersionPolicy": "", // empty
-    "DangerousAcceptAnyServerCertificateValidator": true
+    "DangerousAcceptAnyServerCertificateValidator": true,
+    "HttpHandlerOptions":{
+        "EnableMultipleHttp2Connections": true

```

Comment:
Is it required?
@ibnuda Have you faced with the same problem when the property must be enabled for HTTP/2?

Reply:
hi @raman-m to be honest, i haven't faced the same problem. but if microsoft recommends it, then i think it's better to enable it

Reply:
It appears that all properties of the .NET `SocketsHttpHandler` class ought to be included in Ocelot's `HttpHandlerOptions` class. Currently, the implemented functionality is quite limited, and ideally, Ocelot would offer settings for every `SocketsHttpHandler` property. Indeed, this presents a significant design challenge. 🤔

Reply:
### TODO
- [ ] Create a milestone to implement most of the .NET `SocketsHttpHandler` properties through the `HttpHandlerOptions` class

Reply:
@raman-m 
Do you mean to add all the fields?

Reply:
@EngRajabi Sorry! This is my personal project managing TODO. Please ignore it!
Why not to focus on code review issues and [requested tests](https://github.com/ThreeMammals/Ocelot/pull/2126#pullrequestreview-2271925686)?